app/worker.js

```diff
@@ -62,9 +63,11 @@ class Worker {
   async executeFetchEvent(url, opts) {
     let response = null;
     let waitUntil = [];
+    const request = new Request(url, { redirect: "manual", ...opts });
+    this.setCFHeaders(request);
```

Comment:
Make these two a non-instance method? Like buildRequest(url, opts)?

Reply:
Added a buildRequest method that accepts the express req and a consolidated all the request building logic in server.js. Had to keep the `redirect: 'manual'` behavior in the worker however. What do you think?

Reply:
I'm wondering if the worker spec should take a dependency on `buildRequest` rather than calling `new Request`, or, would your rather continue building the request in the worker? Maybe `executeFetchEvent` should accept the express request?